docs/operation/account_administration.rst

```diff
@@ -43,40 +43,43 @@ perform the following procedures:
    credentials.
 
 
-Administering MetalK8s GUI, Kubernetes(K8S) and Salt API
-********************************************************
+Administering MetalK8s GUI, Kubernetes API and Salt API
+*******************************************************
 
-A fresh install of MetalK8s has a GUI instance with default credentials:
-``admin`` / ``admin``. For more information on how to access the GUI, please
-refer to :ref:`this procedure <quickstart-services-admin-ui>`
+During installation, MetalK8s configures Kubernetes API to accept Basic Auth
+authentication method, with default credentials ``admin`` / ``admin``.
 
-Managing MetalK8s GUI, K8S and Salt API username and password
--------------------------------------------------------------
+Services exposed by MetalK8s, such as
+:ref:`its GUI <quickstart-services-admin-ui>` or
+:ref:`Salt API <quickstart-services-salt>`, rely on Kubernetes API for
+authenticating their users. As such, changing the credentials for
+Kubernetes API default user will also change the credentials required to
+connect to either one of these services.
 
-  .. warning::
+Managing Kubernetes API username and password
+---------------------------------------------
 
-     #. In a typical **Metalk8s** cluster, ``/etc/kubernetes/htpasswd``
-        provides access control credentials to the following 3 components:
-        **MetalK8s GUI**, **K8S** and **Salt API**. Any changes made to this
-        file will directly impact authentication for the 3 mentioned components.
+  .. warning::
 
-     #. The procedures mentioned below must be carried out on every cluster
-        node having an instance of the Kubernetes API server.
+     The procedures mentioned below must be carried out on every control-plane
+     :term:`Node`, or more specifically, a Node bearing the
+     ``node-role.kubernetes.io/master`` label.
 
-#. From a node running an instance of the Kubernetes API server,
-   edit the file ``/etc/kubernetes/htpasswd`` replacing the
-   username and/or password fields as below.
+#. Edit the credentials file located at ``/etc/kubernetes/htpasswd``, replacing
```

Comment:
Do we want to give a better explanation on this file (e.g. give a reference to [the docs](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#static-password-file))? Or, since we plan on using OIDC in the near future (2.4.1 maybe?), should we just stick with this for now?

Reply:
Stick with this and burn HTTP auth with fire ASAP